frontend/Components/Games.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+export interface GameProps {
+	game: string;
+}
+
+export const GamesCheck = (props: GameProps): JSX.Element => <>
```

Comment:
1. Because we're dropping explicit types declarations from linter you can remove the return type on the function.

2. But there is one thing to change:
```
export const GamesCheck: React.FC<GameProps> = (props) => <>
```
`React.FC` is the type for a functional component, and having `GameProps` in the definition of the constant rather than the arrow function makes it slightly more consistent - generally in TS you avoid defining types in arrow functions to make them simpler to parse.